### PR TITLE
Improving pyvips instructions

### PR DESCRIPTION
**Pull request by int-ua**
#### Summary
None

#### Purpose of change
Docs

#### Describe the solution
Concentrated docs, almost no water.

#### Additional context
Co-Authored-by: @AlbertTheTerrible

### CODE REVIEW

**Comment by Fris0uman** [comment]
Why move it somewhere else since it's only relevant to tilesets? Also I'm a bit afraid that putting it in somethign called DEVELOPER_TOOLING will make it look even more like a "scary complicated coding thing" that people that just want to make sprites will recoil from.

**Comment by int-ua** [comment]
Just the usual itch to categorize things into proper places :) ok, I'll think about it. Will be back later.

**Comment by jbytheway** [comment]
Maybe add a link in the opposite direction?

File: doc/TILESET.md [hunk 1]
```diff
@@ -182,17 +182,30 @@ A tilesheet can be an expansion from a mod.  Each expansion tilesheet is a singl
 
 `--palette-copies`: Output copies of tilesheet files quantized to 8bpp palette with `.png8` extension. Intended for external detection if conversion was lossless.
 
-Requires `pyvips` module.
+Requires `pyvips` module, here is how you can install it on Windows:
 
-### Installing pyvips on Windows
```
Comment (Fris0uman) [inline doc/TILESET.md:187]: Doesn't that make it a bit less obvious that this is only for windows? Or

Comment (int-ua) [reply doc/TILESET.md:187]: Is this better?

Comment (Fris0uman) [reply doc/TILESET.md:187]: Yes, nice 👍

**Pull request merged**